dbms/src/Flash/Mpp/newMPPExchangeWriter.h

```diff
@@ -18,11 +18,12 @@
 #include <Flash/Mpp/BroadcastOrPassThroughWriter.h>
 #include <Flash/Mpp/FineGrainedShuffleWriter.h>
 #include <Flash/Mpp/HashPartitionWriter.h>
+#include <Flash/Mpp/MppVersion.h>
 
 namespace DB
 {
 template <class ExchangeWriterPtr>
-std::unique_ptr<DAGResponseWriter> newMPPExchangeWriter(
+std::unique_ptr<DAGResponseWriter> NewMPPExchangeWriter(
```

Comment:
also change the file name?

Reply:
I've renamed func name back to `newMPPExchangeWriter`.